components/forms/w-field.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <component :is="as" class="field" :class="{ 'is-invalid': hasErrorMessage, 'is-disabled': disabled, [$attrs.class || '']: true }" :role="role" v-bind="wrapperAria">
-    <component :is="labelType" v-if="label" class="field-label" :id="labelId" :for="labelFor" :role="valueOrUndefined(labelLevel, 'heading')" :aria-level="valueOrUndefined(labelLevel, labelLevel)">{{ label }}<span v-if="optional" class="pl-8 font-normal text-14 text-gray-500"> (valgfritt)</span></component>
+  <component :is="as" class="relative" :class="{[input.invalid]: hasErrorMessage, [$attrs.class || '']: true}" :role="role" v-bind="wrapperAria">
+    <component :is="labelType" v-if="label" :class="{[l.label]: true, [l.labelValid]: !hasErrorMessage, [l.labelInvalid]: hasErrorMessage}" :id="labelId" :for="labelFor" :role="valueOrUndefined(labelLevel, 'heading')" :aria-level="valueOrUndefined(labelLevel, labelLevel)">{{ label }}<span v-if="optional" :class="input.optional"> (valgfritt)</span></component>
```

Comment:
Small thing but was wondering, would it make sense to keep those optional label classes under `l.optional` instead of `input.optional` for the sake of consistency? Or maybe there's something I'm missing.

Reply:
Ah good point, no you don't miss anything. We've had all classes under `input` before we divided input into `input`, `label` and `helpText`. So we forgot to move optional. I'll fix 👍